<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qCanBox</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="wdg_com_t" name="wdg_com" native="true">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_canbox">
        <property name="text">
         <string>CanBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_canbox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="btn_about">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../qcanbox.qrc">
          <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_door_rl">
        <property name="text">
         <string>DoorRL</string>
        </property>
        <property name="icon">
         <iconset resource="../qcanbox.qrc">
          <normaloff>:/images/rl_door.png</normaloff>:/images/rl_door.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_door_fr">
        <property name="text">
         <string>DoorFR</string>
        </property>
        <property name="icon">
         <iconset resource="../qcanbox.qrc">
          <normaloff>:/images/fr_door.png</normaloff>:/images/fr_door.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_door_fl">
        <property name="text">
         <string>DoorFL</string>
        </property>
        <property name="icon">
         <iconset resource="../qcanbox.qrc">
          <normaloff>:/images/fl_door.png</normaloff>:/images/fl_door.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_door_rr">
        <property name="text">
         <string>DoorRR</string>
        </property>
        <property name="icon">
         <iconset resource="../qcanbox.qrc">
          <normaloff>:/images/rr_door.png</normaloff>:/images/rr_door.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Taho</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="slider_taho">
        <property name="maximum">
         <number>6000</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Battery</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QSlider" name="slider_voltage">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Speed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSlider" name="slider_speed">
        <property name="maximum">
         <number>240</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Fuel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSlider" name="slider_fuel">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Air Temp</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_temp">
          <property name="minimum">
           <number>-40</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_belt">
        <property name="text">
         <string>Belt</string>
        </property>
        <property name="icon">
         <iconset resource="../qcanbox.qrc">
          <normaloff>:/images/belt.png</normaloff>:/images/belt.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_bonnet">
        <property name="text">
         <string>Bonnet</string>
        </property>
        <property name="icon">
         <iconset resource="../qcanbox.qrc">
          <normaloff>:/images/bonnet.png</normaloff>:/images/bonnet.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_tailgate">
        <property name="text">
         <string>Tailgate</string>
        </property>
        <property name="icon">
         <iconset resource="../qcanbox.qrc">
          <normaloff>:/images/tailgate.png</normaloff>:/images/tailgate.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="wdg_dbg_t" name="wdg_dbg"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>wdg_com_t</class>
   <extends>QWidget</extends>
   <header>wdg_com.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>wdg_dbg_t</class>
   <extends>QPlainTextEdit</extends>
   <header>wdg_dbg.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qcanbox.qrc"/>
 </resources>
 <connections/>
</ui>
